x/ccv/provider/keeper/partial_set_security_test.go

```diff
@@ -139,3 +123,94 @@ func TestHandleSetConsumerCommissionRate(t *testing.T) {
 	require.Equal(t, sdk.OneDec(), cr)
 	require.True(t, found)
 }
+
+func TestOptInTopNValidators(t *testing.T) {
```

Comment:
Could we add a test that has multiple validators with the smallest power that is opted in? just to check that edge case

Reply:
Introduced one more validator with power `1` so we have two validators with power `1`. Also added a case where we have `minPowerToOptIn == 0`.